### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by bitterpanda63**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+### Your First Code Contribution
+
+- clone the repository to your local machine
+- run `$ make install` to install dependencies
+- run `$ make build` to build the library
+- run `$ make watch` to watch for changes and rebuild the library
+- run `$ make test` to run tests using tap
+- run `$ make end2end` to run end-to-end tests using tap
+- run `$ make lint` to run ESLint
```
Comment (hansott) [inline .github/CONTRIBUTING.md]: update?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+## Styleguides
+
+### Code format
+
+Code is formatted using [Prettier](https://prettier.io/). See [`.prettierrc`](../.prettierrc) for the configuration.
```
Comment (hansott) [inline .github/CONTRIBUTING.md]: Update?

File: README.md [hunk 3]
```diff
@@ -12,6 +12,9 @@ Firewall autonomously protects your Python applications against:
 
 * 🛡️ [NoSQL injection attacks](https://www.aikido.dev/blog/web-application-security-vulnerabilities)
 * 🛡️ [SQL injection attacks]([https://www.aikido.dev/blog/web-application-security-vulnerabilities](https://owasp.org/www-community/attacks/SQL_Injection))
+* 🛡️ [Command injection attacks](https://owasp.org/www-community/attacks/Command_Injection)
+* 🛡️ [Path traversal attacks](https://owasp.org/www-community/attacks/Path_Traversal)
+* 🛡️ [Server-side request forgery (SSRF)](./docs/ssrf.md)
```
Comment (hansott) [inline README.md:17]: ssrf doesn't exist

File: docs/django.md [hunk 4]
```diff
@@ -1 +1,25 @@
...
+Make sure to set your token in order to communicate with Aikido's servers
+```env
+AIKIDO_TOKEN="AIK_RUNTIME_YOUR_TOKEN_HERE"
+```
+
+- Enabling extra debugging (optional): ```AIKIDO_DEBUG=1```
+- Enabling blocking using an env variable (optional): ```AIKIDO_BLOCKING=1```
+
+## Using gUnicorn
+If you're using gunicorn, please check our docs on that first : [Click Here](./gunicorn.md)
```
Comment (hansott) [inline docs/django.md:22]: Shall we add sections like the Node docs? 😇

File: docs/flask.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+If you're using gunicorn, please check our docs on that first : [Click Here](./gunicorn.md)
+
+
+## Warning: Installing middleware
+When installing middleware make sure to install it like this :
+```python
+from flask import Flask
+app = Flask(__name__)
+...
+app.wsgi_app = my_middleware(app.wsgi_app)
```
Comment (hansott) [inline docs/flask.md:30]: not possible to avoid this, by hooking into Flask?

Comment (bitterpanda63) [reply docs/flask.md:30]: It's the recommended way of setting middleware, if you use more than one middleware you have to do this

Comment (hansott) [reply docs/flask.md:30]: not having to update code > having to update code

Sentry doesn't require that: https://github.com/getsentry/sentry-python/blob/269d96d6e9821122fbff280e6a26956e5ed03c0b/sentry_sdk/integrations/flask.py#L80

https://docs.sentry.io/platforms/python/integrations/flask/

Comment (bitterpanda63) [reply docs/flask.md:30]: Went from 84 lines down to 66 lines ;)

Comment (bitterpanda63) [reply docs/flask.md:30]: Rm'ed

**Pull request merged**